fjord/heartbeat/tests/test_api.py

```diff
@@ -154,6 +154,30 @@ def test_complete_answer(self):
 
         eq_(ans.is_test, False)
 
+    def test_country_unknown(self):
+        """This is a stopgap fix for handling 'unknown' value for country"""
+        survey = SurveyFactory.create()
+
+        data = {
+            'experiment_version': '1',
+            'response_version': 1,
+            'person_id': 'joemamma',
+            'survey_id': survey.name,
+            'flow_id': '20141113',
+            'question_id': '1',
+            'updated_ts': self.timestamp(),
+
+            'question_text': 'ou812?',
+            'variation_id': '1',
+            'country': 'unknown'
+        }
```

Comment:
Can confirm. This matches the docs.

Do you do this a lot? In a different, not-an-emergency time, maybe it would make sense to use FactoryBoy to create api data? As I understand it, it can also generate arbitrary objects too.